otter/convergence/service.py

```diff
@@ -392,6 +391,7 @@ def convergence_succeeded(executor, scaling_group, group_state, resources,
         yield cf_msg('group-status-active',
                      status=ScalingGroupStatus.ACTIVE.name)
     # update servers cache with latest servers
+    now = yield Effect(Func(datetime.utcnow))
```

Comment:
Yes. There should be comment about this. Unfortunately there is dependency between multiple functions calls that should be documented. Will do. Thanks!